.gitignore

```diff
@@ -1 +1,2 @@
 ._*
+*.pem
```

Comment:
Is this for when running the test.sh script?

Reply:
Yes test script and any other runs you would do in context of current directory.